docs/src/get-started/monitor-and-troubleshoot/status.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Git status
```

Comment:
❓is it related to `Git status`? 
Title should be `Platform status`, isn't it?

Reply:
It's more of a metaphorical relationship with `git status`. The idea was each section would relate to a Git command to make it more engaging. There isn't really a `platform status` command. :shrug: